### PR TITLE
coreos-devel/mantle: bump to v0.1.0

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

File: coreos-devel/mantle/mantle-9999.ebuild [hunk 1]
```diff
@@ -20,6 +20,8 @@ DESCRIPTION="Mantle: Gluing CoreOS together"
 HOMEPAGE="https://github.com/coreos/mantle"
 LICENSE="Apache-2"
 SLOT="0"
+# objcopy/split have trouble with our cross-compiled kolet
```
Comment (marineam) [inline coreos-devel/mantle/mantle-9999.ebuild]: You can define a QA_something var to specifically exclude the arm binary. That way the rest still gets stripped.

Comment (marineam) [inline coreos-devel/mantle/mantle-9999.ebuild]: QA_PREBUILT="/usr/lib/kola/*" should do the trick

Comment (mischief) [inline coreos-devel/mantle/mantle-9999.ebuild]: oh, i asked some gentoo folks and they said `STRIP_MASK`

**Comment by marineam** [comment]
LGTM

**Pull request merged**